test/parsing/errors/invalidIndexExpr-detailed.good

```diff
@@ -0,0 +1,7 @@
+─── syntax in invalidIndexExpr.chpl:1 [InvalidIndexExpr] ───
+  Only identifiers or tuples of identifiers are allowed as index expressions.
+  Invalid index expression used here:
+      |
+    1 | for i.x in [1..5] do {
```

Comment:
Did we have the opportunity to underline the `i.x` here?

Reply:
I've given it a try, and it doesn't look like we can -- passing in the relevant AstNode to be highlighted does nothing, presumably because this AstNode doesn't contain relevant location or ID info.